app-directory/.travis.yml

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
I think we should not add this file for now, it is related to the build process for the appd standalone repo.